public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx

```diff
@@ -359,24 +373,79 @@ const EpicTestVariantEditor: React.FC<EpicTestVariantEditorProps> = ({
         />
       )}
 
-      {allowVariantCustomPrimaryCta && (
+      {allowNewsletterSignup && (
+        <FormControl>
+          <RadioGroup
+            value={variant.newsletterSignup ? 'newsletterSignup' : 'buttons'}
+            onChange={onCtasToggleChange}
+          >
+            <FormControlLabel
+              value="newsletterSignup"
+              key="newsletterSignup"
+              control={<Radio />}
+              label="Newsletter signup"
+              disabled={!editMode}
+            />
+            <FormControlLabel
+              value="buttons"
+              key="buttons"
+              control={<Radio />}
+              label="Buttons"
+              disabled={!editMode}
+            />
+          </RadioGroup>
+        </FormControl>
+      )}
+      {variant.newsletterSignup && (
         <div className={classes.sectionContainer}>
           <Typography className={classes.sectionHeader} variant="h4">
-            Buttons
+            Newsletter Signup
           </Typography>
 
-          <EpicTestVariantEditorCtasEditor
-            primaryCta={variant.cta}
-            secondaryCta={variant.secondaryCta}
-            updatePrimaryCta={updatePrimaryCta}
-            updateSecondaryCta={updateSecondaryCta}
-            allowVariantCustomSecondaryCta={allowVariantCustomSecondaryCta}
-            isDisabled={!editMode}
+          <EpicTestNewsletter
+            newsletterSignup={variant.newsletterSignup}
+            updateNewsletterSignup={updateNewsletterSignup}
             onValidationChange={onValidationChange}
-            supportSecondaryCta={allowVariantSecondaryCta}
+            isDisabled={!editMode}
           />
         </div>
       )}
+      {!variant.newsletterSignup && allowVariantCustomPrimaryCta && (
+        <>
+          <div>
+            <div className={classes.sectionContainer}>
+              <Typography className={classes.sectionHeader} variant="h4">
+                Buttons
+              </Typography>
+
+              <EpicTestVariantEditorCtasEditor
+                primaryCta={variant.cta}
+                secondaryCta={variant.secondaryCta}
+                updatePrimaryCta={updatePrimaryCta}
+                updateSecondaryCta={updateSecondaryCta}
+                allowVariantCustomSecondaryCta={allowVariantCustomSecondaryCta}
+                isDisabled={!editMode}
+                onValidationChange={onValidationChange}
+                supportSecondaryCta={allowVariantSecondaryCta}
+              />
+            </div>
+          </div>
+
+          {allowVariantChoiceCards && (
```

Comment:
should this be outside the `!variant.newsletterSignup && allowVariantCustomPrimaryCta` check? I don't think it affects any of the existing configs but it means `allowVariantChoiceCards` is dependent on `allowVariantCustomPrimaryCta` which is slightly confusing

Reply:
This is deliberate because we don't want choice cards to be available if `newsletterSignup` is selected

Reply:
and also it doesn't make sense to have choice cards without the primary cta (ideally the model would better reflect this, but it's too late!)